packages/pigeon/lib/swift_generator.dart

```diff
@@ -44,6 +44,42 @@ class SwiftOptions {
 /// Calculates the name of the codec that will be generated for [api].
 String _getCodecName(Api api) => '${api.name}Codec';
 
+/// Returns a new [Method] using the swift function signature from [func],
+/// ie the name of function e the strings between the semicolons.
+/// Example:
+///   f('void add(int x, int y)') -> ['add', 'x', 'y']
```

Comment:
I don't quite understand the last couple of lines of this docstring.  I think there is a typo with "function e the strings".  The example also seems to show this function returning a list of strings which isn't the case.  It also refers to function `f`, is that supposed to be `_methodWithSwiftFunction`?

Reply:
Improved